code/sorting/src/pancake_sort/pancake_sort.cpp

```diff
@@ -0,0 +1,99 @@
+/*
+    Pancake Sorting Algorithm
+*/
+#include<iostream>
+#include<vector>
+
+// Print the sorted array
+
+void print_ans(std::vector<int> v , int n)
+{
+    std::cout<<"Sorted Array\n";
+    
+    for(int i=0 ; i < n ; ++i)
+   {
+        std::cout<<v[i]<<" ";  
+   }
+}
+
+// finding the index of largest number in the given array size
+
+int find_index(std::vector<int> v , int size)
```

Comment:
All vectors should be passed as const references if they are not being modified.